Classes/WPEditorToolbarButton.m

```diff
@@ -1,19 +1,25 @@
 #import "WPEditorToolbarButton.h"
+#import <WordPress-iOS-Shared/WPStyleGuide.h>
+
+static NSString* const CircleLayerKey = @"circleLayer";
+static int TouchAnimationCircleRadius = 15;
+static int TouchAnimationCircleRadiusiPad = 17;
```

Comment:
Since these are used for radius, and `UIBezierPathbezierPathWithArcCenter:radius:startAngle:endAngle:clockwise` takes a `CGFloat` for its radius arg, these should probably be declared as `CGFloat`s. 
But fwiw, foundation types like `NSInteger` would generally be preferred over C types like `int`